docs/core/logging.md

```diff
@@ -615,73 +496,153 @@ To append an additional key in your logs, you can use the `LoggingUtils.appendKe
+
+    === "Function1.java"
+    
+        ```java
+        LOGGER.info("Processing {}", entry("order", order));
+        ```
+
+    === "Order.java"
+
+        ```java hl_lines="5"
+        public class Order {
+            // ...        
+
+            @Override
+            public String toString() {
+                return "Order{" +
+                        "orderId=" + id +
+                        ", amount=" + amount +
+                        ", date='" + date + '\'' +
+                        ", customerId=" + customerId +
+                        '}';
+            }
+        }
+        ```
+    
+    === "CloudWatch Logs Function1"
+    
+        ```json hl_lines="3 7"
+        {
+            "level": "INFO",
+            "message": "Processing order=Order{orderId=23542, amount=459.99, date='2023-12-01T14:49:19.018Z', customerId=328496}",
+            "service": "payment",
+            "timestamp": "2023-12-01T14:49:19.293Z",
+            "xray_trace_id": "1-6569f266-4b0c7f97280dcd8428d3c9b5",
+            "order": {
+                "orderId": 23542,
+                "amount": 459.99,
+                "date": "2023-12-01T14:49:19.018Z",
+                "customerId": 328496
+            }
+        }
+        ```
+    
+    You can also combine structured arguments with non structured ones. For example:
+
+    === "Function2.java"
+        ```java
+        LOGGER.info("Processing order {}", order.getOrderId(), entry("order", order));
+        ```
+
+    === "CloudWatch Logs Function2"
+        ```json
+        {
+                "level": "INFO",
+                "message": "Processing order 23542",
+                "service": "payment",
+                "timestamp": "2023-12-01T14:49:19.293Z",
+                "xray_trace_id": "1-6569f266-4b0c7f97280dcd8428d3c9b5",
+                "order": {
+                    "orderId": 23542,
+                    "amount": 459.99,
+                    "date": "2023-12-01T14:49:19.018Z",
+                    "customerId": 328496
+                }
+            }
+        ```
+
+** Using MDC **
+
+Mapped Diagnostic Context (MDC) is essentially a Key-Value store. It is supported by the [SLF4J API](https://www.slf4j.org/manual.html#mdc){target="_blank"},
+[logback](https://logback.qos.ch/manual/mdc.html){target="_blank"} and log4j (known as [ThreadContext](https://logging.apache.org/log4j/2.x/manual/thread-context.html){target="_blank"}). You can use the following standard:
+
+`MDC.put("key", "value");`
+
+???+ warning "Custom keys stored in the MDC are persisted across warm invocations"
+    Always set additional keys as part of your handler method to ensure they have the latest value, or explicitly clear them with [`clearState=true`](#clearing-state).
+
+
+### Removing additional keys
+
+You can remove additional keys added with the MDC using `MDC.remove("key")`. 
 
```

Comment:
"Typically, you'd remove additional keys at the end of your request handler, so they do not apply to subsequent requests"